requirements.txt

```diff
@@ -61,7 +61,7 @@ redis==2.10.6
 requests==2.20.0
 responses==0.9.0
 six==1.11.0
-smpplib==0.1
+smpplib==2.0
```

Comment:
We have not started, yet, but we should do.

So please start adding a note here,
https://github.com/privacyidea/privacyidea/blob/master/READ_BEFORE_UPDATE.md
so that we do not forget, that updates need to watch out in case of smpp.

Reply:
added in fd8b091d